apps/src/gamelab/GameLab.js

```diff
@@ -262,6 +262,11 @@ GameLab.prototype.init = function (config) {
     // to starting code by levelbuilders will be shown.
     config.ignoreLastAttempt = config.embed;
 
+    if (this.studioApp_.isUsingBlockly()) {
+      // Custom blockly config options for game lab jr
+      config.valueTypeTabShapeMap = { Sprite: 'angle' };
```

Comment:
Is this the best place to make this association?  What would it take to not need legacy-style block definitions below?

Reply:
It's a global Blockly option, I'd rather set it when we inject Blockly rather than somewhere down the line.

Avoiding the legacy-style block definitions is an unrelated issue, it's just that variable blocks have number unique settings and properties that I didn't want to clutter up the block creation code with.